buildscripts/zfs-driver/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -ex
+
+/usr/local/bin/zfs-driver
```

Comment:
This entire file can be managed from within Dockerfile.
This is same as my earlier comments. Let us reduce the dependency of build, image & others to just Makefile & Dockerfile.

Reply:
will take care in next PR.